conan/tools/cmake/cmakedeps/templates/__init__.py

```diff
@@ -17,13 +17,9 @@ def __init__(self, cmakedeps, require, conanfile, find_module_mode=False):
     def pkg_name(self):
         return self.conanfile.ref.name + self.suffix
 
-    @property
-    def target_namespace(self):
-        return self.get_target_namespace(self.conanfile) + self.suffix
-
     @property
     def global_target_name(self):
```

Comment:
Given the changes we are introducing, I would rename this method to `root_target_name`